tests/io/vasp/test_help.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import io
+from unittest.mock import patch
+
+import pytest
+import requests
+
+from pymatgen.io.vasp.help import VaspDoc
+
+try:
+    from bs4 import BeautifulSoup
+except ImportError:
+    BeautifulSoup = None
+
+
+try:
+    website_down = requests.get("https://www.vasp.at", timeout=60).status_code != 200
```

Comment:
maybe a 5 second timeout would be better to not overly delay tests? not matter if the test is sometimes skipped on slow connections

Reply:
Great point. Would change this.